python/tk_desktop/desktop_window.py

```diff
@@ -125,10 +125,24 @@ def __init__(self, parent=None):
                 except Exception:
                     pass
 
-        # populate user menu
+        # populate user Menu
         self.user_menu = QtGui.QMenu(self)
         name_action = self.user_menu.addAction(current_user["name"])
         url_action = self.user_menu.addAction(connection.base_url.split("://")[1])
+
+        # Build a menu with all the commands
+        commands = engine.commands
+        if commands:
+            self.user_menu.addSeparator()
+            for name, command in commands.iteritems():
+                props = command["properties"]
+                action = QtGui.QAction(self)
+                action.setObjectName(props["short_name"])
+                action.setText(name)
+                action.setToolTip(props.get("description", "No description available."))
```

Comment:
I think it's probably better to just omit the tooltip rather than having it explicit say "no info available".